rules/github-sprint-templates/rule.md

```diff
@@ -1,5 +1,6 @@
 ---
 type: rule
+archivedreason: Replaced by [Do you create a Sprint Review/Retro email?](/rules/sprint-review-retro-email/)
```

Comment:
I don't think that the reason is clear.
Consider adding some of the reasons from following the rule in anger

> The problems with GitHub Issues:
  ❌ It makes the backlog a mess
  ❌ Need to have a separate private repo if your repo is public ( Only apply to if you want your Sprint Review private)
  ❌ It's not what GitHub Issue designed for
  ❌ The scope (It should be project instead of repo)

Reply:
+1 this, if I recall correctly there was a huge discussion about this topic involving @bradystroud and @gordon that resulted in documented reasons for each possible solution.

If that artifact exists, then I would link to it.

Reply:
Good call, I didn't think of that 🙂

Reply:
Archived reason updated.